lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtRouting.java

```diff
@@ -136,7 +136,12 @@ public PtRoutingResponse findPtRoute(PtRoutingRequest request) {
         );
         // ghRequest.setBlockedRouteTypes(request.getRoutingParameters().excludedPtModes);//FIXME generalize this
```

Comment:
Do we still need this?

Reply:
To be fair, currently we don't need anything of this 🫣 But yes, I will delete that and we will see once we actually use this if we would need to exclude certain public transport modes.